lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -136,7 +136,7 @@ def normalize_id
         # Special processing for loc ids for backward compatibility.  IDs may be in the form 'n123' or 'n 123'.  This adds
         # the <blank> into the ID to allow it to be found as the object of a triple in the graph.
         def loc_id
-          loc_id = URI.unescape(id)
+          loc_id = id
```

Comment:
just change to `loc_id = id.dup` to avoid the "can't mutate frozen string" errors that we're seeing in CI failure currently -- the unescape had the side-effect of creating a string copy too, without the unescape I guess we need to create a copy manually because subsequent code will mutate it for some reason.